app/models/tp-filter.js

```diff
@@ -33,9 +37,9 @@ export default Model.extend(Validations, {
   tpid: attr('string'),
   tenant: attr('string'),
   customId: attr('string'),
-  filterType: attr('string'),
-  filterFieldName: attr('string'),
-  filterFieldValues: attr('string'),
   activationInterval: attr('string'),
   createdAt: attr('date'),
+  type: attr('string'),
```

Comment:
API does not return `type` attribute, The correct name of this attr is cgType. See API doc https://github.com/cgrates-web/cgrates_web_jsonapi/blob/master/API.md#cgrateswebjsonapitpfiltercontrollershow